file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [15-May-2025 21:28:15 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_quizes`
                        DROP INDEX `FK_quiz_category`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_quizes__FK_quiz_category&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_quizes__FK_quiz_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(221): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_quizes`
                        DROP INDEX `FK_quiz_category` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:15 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_questions`
                        DROP INDEX `FK_question_category`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_questions__FK_question_category&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_questions__FK_question_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(227): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_questions`
                        DROP INDEX `FK_question_category` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:15 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_answers`
                        DROP INDEX `FK_answer_question`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_answers__FK_answer_question&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_answers__FK_answer_question.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(233): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_answers`
                        DROP INDEX `FK_answer_question` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:34 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_quizes`
                        DROP INDEX `FK_quiz_category`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_quizes__FK_quiz_category&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_quizes__FK_quiz_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(221): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_quizes`
                        DROP INDEX `FK_quiz_category` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:34 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_questions`
                        DROP INDEX `FK_question_category`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_questions__FK_question_category&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_questions__FK_question_category.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(227): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_questions`
                        DROP INDEX `FK_question_category` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:34 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE `wp_aysquiz_answers`
                        DROP INDEX `FK_answer_question`</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DROP INDEX &quot;wp_aysquiz_answers__FK_answer_question&quot; | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such index: wp_aysquiz_answers__FK_answer_question.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(233): WP_SQLite_DB->query('ALTER TABLE `wp...')
#2 /wordpress/wp-content/plugins/quiz-maker/includes/class-quiz-maker-activator.php(481): Quiz_Maker_Activator::activate()
#3 /wordpress/wp-content/plugins/quiz-maker/quiz-maker.php(64): Quiz_Maker_Activator::ays_quiz_update_db_check()
#4 /wordpress/wp-includes/class-wp-hook.php(324): activate_quiz_maker(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_quiz-m...', false)
#8 /internal/eval.php(20): activate_plugin('quiz-maker/quiz...')
#9 {main}</pre>
 for query ALTER TABLE `wp_aysquiz_answers`
                        DROP INDEX `FK_answer_question` made by activate_plugin, do_action('activate_quiz-maker/quiz-maker.php'), WP_Hook->do_action, WP_Hook->apply_filters, activate_quiz_maker, Quiz_Maker_Activator::ays_quiz_update_db_check, Quiz_Maker_Activator::activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[15-May-2025 21:28:53 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121
[15-May-2025 21:29:01 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121
[15-May-2025 21:29:10 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"quote-curator"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/quote-curator.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/quote-curator.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/quote-curator.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/quote-curator.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
